atomic_reactor/plugins/pre_pyrpkg_fetch_artefacts.py

```diff
@@ -26,13 +29,59 @@ def __init__(self, workflow):
         constructor
 
         :param workflow: DockerBuildWorkflow instance
-        :param command: str, command to use to get artefacts (e.g. 'make sources')
-                             it is executed in cloned git repo
+        :type workflow: atomic_reactor.inner.DockerBuildWorkflow
         """
         # call parent constructor
         super(DistgitFetchArtefactsPlugin, self).__init__(workflow)
         self.command = self.workflow.conf.sources_command
 
+    def _collect_exploded_sources_files(self, build_dir_path: Path) -> List[Path]:
+        """Collect the fetched exploded sources.
+
+        :param build_dir_path: from which directory to find the exploded sources file.
+        :type build_dir_path: pathlib.Path
+        """
+        source_repos = build_dir_path / 'source-repos'
+        filenames = []
+        if not source_repos.exists():
+            return filenames
+        with source_repos.open('r', encoding='utf-8') as f:
+            for line in f:
+                git_ref = line.rstrip().split()[-1]
+                filenames.append(next(build_dir_path.glob(f'*-{git_ref}.tar.gz')))
```

Comment:
Is it safe to glob with just the git ref? We don't know what the dist-git repo may contain. Could we instead return the filename exactly as it would be constructed by rhpkg?

Reply:
@chmeliik almost rewrite this method. Add some checks on the file names.

Reply:
+1, I like this approach more, just a minor detail about potential inconsistency with the way rhpkg does things